airflow/utils/isis2json/argparse.py

```diff
@@ -0,0 +1,2353 @@
+# -*- coding: utf-8 -*-
```

Comment:
Este não seria o caso de importar a biblioteca isis2json no lugar de copiar e incluir no projeto?

Reply:
Concordo, segue link do projeto: 

https://github.com/bireme/isis2json

Reply:
Infelizmente não é possível @robertatakenaka e @jamilatta. Desde o projeto de migração a versão do isis2json vem sendo modificada. Desta última vez foi necessário alterar a forma de output do formato JSON para uma única linha já que o bash processor do airflow pega última linha do stdout:

https://github.com/scieloorg/opac-airflow/blob/6af97e0076151706242a4e2787f143d0323d0600/airflow/utils/isis2json/isis2json.py#L295-L299

Reply:
@joffilyfe 

Ruim essa abordagem! 

Esses ajustes/melhorias não são passíveis de PRs para o repositório da Bireme?

Reply:
Não, ela são inerentes aos nossos projetos.

Reply:
@joffilyfe 

Um fork nosso?

Reply:
Pode até ser em um futuro posterior. No momento nós combinamos em fazer copy paste nos códigos e seguir em frente.

Reply:
@joffilyfe no entanto não seria bom haver um repositório com as atualizações do isis2json e aí ela possa ser importada?

Reply:
Haverá se fizer sentido mantermos essas alterações. As alterações de agora fazem sentido no escopo do projeto de espelhamento. As alterações que temos nesse repositório dizem respeito a quais caracteres são interpretados como listas (o que fazia o xylose quebrar) e também temos alterações no output dos dados.

As modificações no modo como os caracteres são lidos (https://github.com/scieloorg/document-store-migracao/commit/a303dbf171a8ae4a37655fa17bf1a0ebe6c1b3d4#diff-e36315f74690e1a1d64945c67fcbfbb7) pode trazer um impacto não calculado para os processos que já estão em funcionamento (isis2mongo). Tenho uma forte inclinação a não modificar a biblioteca até termos a ideia validada e termos fôlego para eventuais manutenções.

Reply:
Entendo os argumentos em favor de criarmos um fork do isis2mongo onde aplicaríamos as modificações de maneira mais organizada. Acho que podemos fazer isso nas próximas semanas. Podemos pensar que se trata de uma dívida técnica que estamos contraindo agora a fim de acelerarmos a integração dos sistemas, e que será paga quando for conveniente.